WcaOnRails/app/models/event.rb

```diff
@@ -54,4 +54,12 @@ def fewest_moves?
   def multiple_blindfolded?
     self.id == "333mbf" || self.id == "333mbo"
   end
+
+  def serializable_hash(options = nil)
```

Comment:
I'm not sure what these are (Googling talks about an ActiveModel alternative to `to_json`), but without tests they'll break whenever columns are added, renamed or removed.